Lombiq.Hosting.MediaTheme.Targets/Readme.md

```diff
@@ -0,0 +1,37 @@
+# Lombiq Hosting - MediaTheme MSBuild Targets
+
+[![Lombiq.Hosting.MediaTheme.Targets NuGet](https://img.shields.io/nuget/v/Lombiq.Hosting.MediaTheme.Targets?label=Lombiq.Hosting.MediaTheme.Targets)](https://www.nuget.org/packages/Lombiq.Hosting.MediaTheme.Targets/)<!-- #spell-check-ignore-line -->
+
+## About
+
+Provides automatic media theme package creation in the project. This way you don't have to manage changes in your media theme recipe, or folder structure. The targets reuses the media-theme-deployer dotnet tool will be automatically installed locally to the project, then it will create the deployment zip file. Please read the [Lombiq.Hosting.MediaTheme Readme](https://github.com/Lombiq/Hosting-Media-Theme/blob/dev/Readme.md) for more information about this deployer tool.<!-- #spell-check-ignore-line -->
```

Comment:
Why do have to ignore this line?

Reply:
Here, the next line starts without an empty line in between. You need to put text either on the same line, or separate it by an empty line.